common/defs/support.json

```diff
@@ -83,9 +83,8 @@
       "eth:ETC:61": "1.11.3",
       "eth:ETH:1": "1.11.3",
       "eth:MATIC:137": "1.11.3",
-      "eth:tETH:3": "1.6.2",
-      "eth:tETH:4": "1.11.3",
-      "eth:tETH:5": "1.11.3",
+      "eth:tSEP:11155111": "1.11.3",
+      "eth:tHOL:17000": "1.11.3",
```

Comment:
you just added `tSEP` and `tHOL` in this PR. the correct support version is 1.13.0 (next legacy release)